src/common/dns_resolve.cc

```diff
@@ -196,6 +196,11 @@ int DNSResolver::resolve_ip_addr(CephContext *cct, const string& hostname,
   if (r < 0) {
     return r;
   }
+
```

Comment:
nit, might want to remove the empty lines.

Reply:
@tchaikov the empty line has been removed